common/recommender/Baseline.js

```diff
@@ -109,10 +109,12 @@ class Baseline {
       newScore *= 0.2;
     }
 
-    // Boost boomarks even if they have low score or no images giving a
-    // just-bookmarked page a near-infinite boost
-    if (features.bookmarkAge) {
-      newScore += BOOKMARK_AGE_DIVIDEND / features.bookmarkAge;
+    if (this.options.experiments && this.options.experiments.bookmarkScreenshots) {
+      // Boost boomarks even if they have low score or no images giving a
+      // just-bookmarked page a near-infinite boost
+      if (features.bookmarkAge) {
+        newScore += BOOKMARK_AGE_DIVIDEND / features.bookmarkAge;
+      }
```

Comment:
@jaredkerim do we still want this change? I know there was some discussion, but not sure what was decided.